### PR TITLE
test

### PR DESCRIPTION
**Pull request by zetafish**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zetafish/ReviewNinja-Welcome/pull/2" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-WelcomeThis is a wonderfull thing
```
Comment (zetafish) [inline ReadMe.md]: Iets meer context graag `kaboom`

Comment (zetafish) [inline ReadMe.md]: !fix

Comment (zetafish) [inline ReadMe.md]: !completed

**Pull request merged**